src/EnergyPlus/ElectricBaseboardRadiator.cc

```diff
@@ -471,40 +440,41 @@ namespace ElectricBaseboardRadiator {
             ShowFatalError(state, std::string{RoutineName} + cCurrentModuleObject + "Errors found getting input. Program terminates.");
         }
 
-        for (BaseboardNum = 1; BaseboardNum <= NumElecBaseboards; ++BaseboardNum) {
+        for (int BaseboardNum = 1; BaseboardNum <= NumElecBaseboards; ++BaseboardNum) {
```

Comment:
Looks like the index is never used except to lookup the baseboard instance, so this is an opportunity to just loop over the objects and avoid the indexing:

```
for (auto const & elecBaseboard : state.dataElectBaseboardRad->ElecBaseboard) {
```

One great benefit of doing it this way is that when we move from Array1D holders for the objects and go to plain vectors, we don't have to worry about this one having an index problem starting at 1.  Just one less possible future problem.

Reply:
I missed that opportunity while focusing on CppCheck. I'll make this change.

Reply:
The compiler doesn't like that change.

Reply:
OK, it was the const that the compiler didn't like.